test/e2e/utils/utils.go

```diff
@@ -222,9 +222,12 @@ func TestWebhookServerTLS(f *framework.Framework, serviceName string) error {
 				{
 					Args: []string{
 						"--openssl=/usr/bin/openssl",
-						"-p",
+						"--mapping iana",
```

Comment:
It looks the way how this list gets to `testssl.sh` command line requires:
```suggestion
						"--mapping=iana",
```

Reply:
Actually there is no equals required between the arguments.

Reply:
@huornlmj Hmm OK, I see. Anyway, as it is now the test case fails with:
```
 /usr/local/bin/testssl.sh: unrecognized option "--mapping iana"
```

Reply:
I wonder why that isn't working.  Here's it running on my system:
```console
$ ./testssl.sh --mapping iana -s -f -p -P -U --html https://silvop01

###########################################################
    testssl.sh       3.1dev from https://testssl.sh/dev/
    (730c758 2022-08-02 13:28:56)

      This program is free software. Distribution and
             modification under GPLv2 permitted.
      USAGE w/o ANY WARRANTY. USE IT AT YOUR OWN RISK!

       Please file bugs @ https://testssl.sh/bugs/

###########################################################

 Using "OpenSSL 1.0.2-chacha (1.0.2k-dev)" [~183 ciphers]
 on silvjcullig01:./bin/openssl.Linux.x86_64
 (built: "Jan 18 17:12:17 2019", platform: "linux-x86_64")


 Start 2022-08-18 19:18:34        -->> 10.237.215.194:443 (silvop01) <<--
```

Reply:
> I wonder why that isn't working. Here's it running on my system:

I have tested that this suggestion works:

```
"--mapping",
"iana",
```

Reply:
@huornlmj can you make the change so that the tests pass?